language/analyze.js

```diff
@@ -32,7 +32,7 @@ function analyzeSentimentOfText (text) {
   // Detects the sentiment of the document
   document.detectSentiment()
     .then((results) => {
-      const sentiment = results[0];
+      const sentiment = results[1].documentSentiment;
```

Comment:
Is it just that the first result is missing sentiment?  I'm seeing that not all the entities returned have sentiment.